code/Weapons/Components/ExplosiveWeaponComponent.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Grubs;
+
+[Prefab]
+public partial class ExplosiveWeaponComponent : WeaponComponent
+{
+	[Prefab, Net]
+	public string ExplosivePrefabPath { get; set; }
```

Comment:
Is there no way to reference another prefab within a component? Using path is fine but not really ideal.

Reply:
Nope, prefabs cannot contain other prefabs at the moment.